### PR TITLE
FASTMOVING-1645 Exclude known failure from LTP failure snippet tool

### PR DESCRIPTION
**Pull request by idorax**

### CODE REVIEW

File: distribution/ltp-upstream/include/ltp_log_parser.py [hunk 1]
```diff
@@ -153,15 +157,41 @@ def parse(file_handle):
...
           file=sys.stderr)
     print("e.g.",
           file=sys.stderr)
     print("o dump failed test cases",
           file=sys.stderr)
     print("       {} -F /tmp/foo.log".format(prog),
           file=sys.stderr)
+    print("       {} -f /tmp/KNOWNISSUES -F /tmp/foo.log".format(prog),
+          file=sys.stderr)
+    print("       {} -v -f /tmp/KNOWNISSUES -F /tmp/foo.log".format(prog),
```
Comment (rasibley) [inline distribution/ltp-upstream/include/ltp_log_parser.py]: Did you mean to include this line twice, once with -f and again with -v -f ?

Comment (idorax) [reply distribution/ltp-upstream/include/ltp_log_parser.py]: `-v` means `verbose`. I updated the script usage to make it clear. Please refer to:
* https://gitlab.cee.redhat.com/huanli/pluto/blob/sandbox-FASTMOVING-1645/PR473/follow-up/00.md

**Comment by idorax** [comment]
Hi @rasibley , a beaker job is done, its job id is `4092919`. And I also filed a [summary][101] to explain the test result is expected, please help to review a again.

[101]: https://gitlab.cee.redhat.com/huanli/pluto/blob/sandbox-FASTMOVING-1645/PR473/summary.md

File: distribution/ltp-upstream/include/ltp_log_parser.py (hunk 1 above)
Comment (rasibley) [reply distribution/ltp-upstream/include/ltp_log_parser.py]: Great thanks for adding that!

**Comment by rasibley** [comment]
Based on the test jobs, I think we're good to merge, thanks @idorax for your work on this!

**Pull request merged**